client/my-sites/google-my-business/index.js

```diff
@@ -68,31 +50,63 @@ export default function( router ) {
 			siteSelection,
 			selectLocation,
 			navigation,
-			makeLayout,
+			makeLayout
 		);
 
-		router(
-			'/google-my-business/stats',
-			redirectLoggedOut,
-			siteSelection,
-			sites,
-			makeLayout,
-		);
+		router( '/google-my-business/stats', redirectLoggedOut, siteSelection, sites, makeLayout );
 
 		router(
 			'/google-my-business/stats/:site',
 			redirectLoggedOut,
 			siteSelection,
 			stats,
 			navigation,
-			makeLayout,
+			makeLayout
+		);
+
+		router(
```

Comment:
We cannot move this route definition into the `if ( config.isEnabled( 'google-my-business' ) ) {` because of this:
https://github.com/Automattic/wp-calypso/pull/23741/files#diff-3fb2cdece814c85c3785ab6d1ad8f9c9R93